content/en/docs/setup/install/multicluster/index.md

```diff
@@ -83,692 +90,20 @@ below may have to be altered based on your choice of CA.
 ## Install Istio
 
 The steps for installing Istio on multiple clusters depend on your
-requirements for network and control plane topology. This section
-illustrates the options for two clusters. Meshes that span many clusters may
-employ more than one of these options. See
-[deployment models](/docs/ops/deployment/deployment-models) for more
-information.
-
-{{< tabset category-name="install-istio" >}}
-
-{{< tab name="Multi-Primary" category-value="multi-primary" >}}
-
-The steps that follow will install the Istio control plane on both `cluster1` and
-`cluster2`, making each a {{< gloss >}}primary cluster{{< /gloss >}}. Both
-clusters reside on the `network1` network, meaning there is direct
-connectivity between the pods in both clusters.
-
-Each control plane observes the API Servers in both clusters for endpoints.
-
-Service workloads communicate directly (pod-to-pod) across cluster boundaries.
-
-{{< image width="75%"
-    link="multi-primary.svg"
-    caption="Multiple primary clusters on the same network"
-    >}}
+requirements for network and control plane topology. Choose the steps
```

Comment:
Link to deployment models concepts page here.

Reply:
I already link below in the tip. Should I move it here?

Reply:
This content has moved to the "Before you begin" page ... and I moved up the tip so it's more clear.